dv_utils/client.py

```diff
@@ -38,7 +92,7 @@ def get_data(self, user_id: str, rdf_format: str = "turtle"):
         If the format is turtle (default), the returned value is an rdflib graph.
         """
         raw_data = self.request(
-            f"/clients/{self.settings.client_id}/applications/{self.settings.app_id}/activeUsers/{user_id}/data"
+            f"/clients/{self.settings.collaboration_space_owner_id}/applications/{self.settings.collaboration_space_id}/activeUsers/{user_id}/data"
```

Comment:
we are planning on retiring this endpoint. You can keep it for now if everything works, but we are planning to only use `collaborationSpaces/{space_id}/users/{user_id}/appData/{path}`.

The API will prepend the configured `podFolder` from the db to the `path` variable.

Reply:
This was existing code. I prefer not to touch this :)  But of course please do...